adamacs/behavior.py

```diff
@@ -113,8 +93,50 @@ class TrialVariable(dj.Part):
 		variable_value: varchar(2000)
 		"""
 
+@schema
+class BehaviorIngest(dj.Imported):
+	definition = """
+	-> session.Recording
+	"""
+  @property
+  def key_source(self): #opional means of subsetting or reordering keys worth ingesting
+  	good_records = session.Recording.fetch('KEY') #& session.Qualitycontrol
+  	return good_records
+
+	def make(self, key): # reading bpod data to populate
+		# could model dir navigation after element_array_ephys
+		# which uses config file for root dir and csv for relative paths
+		# https://github.com/datajoint/workflow-array-ephys/blob/main/workflow_array_ephys/paths.py
+		bpod_root_dir = pathlib.Path(get_beh_root_dir(key))
+		bpod_sess_dir = pathlib.Path(get_beh_sess_dir(key))
+		bpod_dir = find_full_path(bpod_root_dir,bpod_sess_dir)
+
+		for file in os.listdir(bpod_dir): if filename.endswith(".mat"):
+			trial_info = load_bpod_matfile(key, bpod_dir + file)
+		schema.Trial.insert(trial_info,ignore_extra_fields=True)
+		schema.Event.insert(trial_info,ignore_extra_fields=True)
+
+# --------------------- HELPER LOADER FUNCTIONS -----------------
+
+# see full example here:
+# https://github.com/mesoscale-activity-map/map-ephys/blob/master/pipeline/ingest/behavior.py
+
+def load_bpod_matfile(key, matlab_filepath):
+	"""
+	Loading routine for behavioral file, bpod .mat
+	"""
+	#Loading the file
+	SessionData = spio.loadmat(matlab_filepath.as_posix(),
+                            squeeze_me=True, struct_as_record=False)['SessionData']
+
+	# Add to dict for insertion
+  for trial in range(SessionData.nTrials):
+  	trial_info['start_time'] = SessionData.RawData.OriginalEventTimestamps[trial]
```

Comment:
This block doesn't quite make sense to me, I think you meant for this to be place holder, to demonstrate the steps? If so let's comment this out

Reply:
Sure, I'll comment this out and move both the table and the function skeleton to an embedded module.